### ISSUE TITLE
line numbers thwart use of print out

### ISSUE THREAD

**Issue by m-stgt** [gomi42/FocalMaster#3]
My goal was to prepare a program barcode. So I printed the program in question (printer set to MAN) and transferred via Windows clipboard the print out to "FocalMaster", tab "FOCAL source". Result:

- Unknown statement "01•LBL", line 1, " 01•LBL "%T""
- Unexpected parameter "1/X", line 2, " 02 1/X"
- Unexpected parameter "%", line 3, " 03 %"
- Unexpected parameter "1/X", line 4, " 04 1/X"
- Unexpected parameter ".END.", line 5, " 05 .END."

So the print out of a virtual HP41 is not suitable as input.

**Comment by gomi42** [comment]
Correct, the editor does not support line numbers right now. Problem: how to distinguish line numbers from number values in case line numbers are optional?

**Comment by m-stgt** [comment]
In the description, section 'Focal Compiler', second paragraph, you affirm _"Strong error checking is done during the compilation."_ Undoubtedly you will find **strong criterions** if a listing includes line numbers or not. Maybe even a trace mode print out could be parsed. (If I grasp such a listing, an application may also analyse it.)
